### PR TITLE
readme layout update

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

**Review by tomzy-0 — APPROVED** [review]

**Comment by tomzy-0** [comment]
@PLangowski this should target `master` or `develop`?

File: docs/cpu.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The `cpu` command allows you to log in to a remote system (in this case RTE) and
+see some of the files from your local system. This allows you to execute
+programs installed on your local system in the remote environment. Another use
+case is executing `minicom` on RTE and storing the logs on your host PC without
+having to `scp` them and without worrying about RTE running out
+of space. This use case is demonstrated in this document.
+
+## Prerequisites
+
+* RTE with system version `0.7.4-rc5` or later connected to your local network
```
Comment (tomzy-0) [inline docs/cpu.md]: @PLangowski let's add here v0.7.4 so the full version that we are about to release

File: docs/cpu.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# CPU example usage
+
+This document describes an example use of the [cpu](https://github.com/u-root/cpu)
+command.
+
+## Introduction
+
+The `cpu` command allows you to log in to a remote system (in this case RTE) and
```
Comment (tomzy-0) [inline docs/cpu.md:8]: @PLangowski The quality of this description could be a lot better. Can we not improve it? I think the main advantage of cpu is not that we can `see some files` from host PC. Please provide better description.

File: docs/cpu.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Prerequisites
+
+* RTE with system version `0.7.4-rc5` or later connected to your local network
+* Some bootable platform connected to the RTE via serial
+* The `cpu` command installed on your host PC
+
+## Using cpu
+
+This section describes how to use the `cpu` command to execute `minicom` on RTE
```
Comment (tomzy-0) [inline docs/cpu.md:24]: Let's point here to some advantages, e.g. we do not need to worry about ending storage space on RTE.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
 
+# Meta-RTE Yocto BSP documentation
+
+* [BSP layers description](docs/layer-description.md)
+
+* [Development process description](docs/development.md)
+
+* [Release process description](docs/release.md)
+
+* [Instruction of setting the MAC address in U-Boot](docs/rte-mac-setup.md)
```
Comment (tomzy-0) [inline README.md:12]: @PLangowski let's add here also a link for cpu notes.

**Review by tomzy-0 — CHANGES_REQUESTED** [review]
@PLangowski I left a review.

File: docs/cpu.md (hunk 3 above)
Comment (PLangowski) [reply docs/cpu.md:24]: I already mentioned this in the introduction. Should I repeat it?

File: docs/cpu.md (hunk 1 above)
Comment (PLangowski) [reply docs/cpu.md]: done e2e288c2cfe632e4be4be87116ed66b448439d17

File: docs/cpu.md (hunk 2 above)
Comment (PLangowski) [reply docs/cpu.md:8]: done e2e288c2cfe632e4be4be87116ed66b448439d17
Is this ok?

File: docs/cpu.md (hunk 3 above)
Comment (PLangowski) [reply docs/cpu.md:24]: i mentioned it again e2e288c2cfe632e4be4be87116ed66b448439d17

File: README.md (hunk 4 above)
Comment (PLangowski) [reply README.md:12]: done 4d38fd7f2dd3fab263b08ce6fce853abf4b02870

File: docs/cpu.md (hunk 2 above)
Comment (tomzy-0) [reply docs/cpu.md:8]: A lot better. Thanks

**Review by tomzy-0 — APPROVED** [review]

**Comment by tomzy-0** [comment]
> @PLangowski this should target `master` or `develop`?

@PLangowski I think we need this on develop

**Pull request merged**